<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectraDialog</class>
 <widget class="QDialog" name="SpectraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectra Visualization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="11">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QPushButton" name="push_save">
     <property name="text">
      <string>&amp;Save Image...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="push_customize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Customi&amp;ze &gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="10">
    <widget class="QGroupBox" name="gb_customize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>207</height>
      </size>
     </property>
     <property name="title">
      <string>Customize</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Calculated Spectra:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="push_colorCalculated">
        <property name="text">
         <string>Set Color...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="cb_labelPeaks">
        <property name="text">
         <string>&amp;Label peaks</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Imported Spectra:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="push_colorImported">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set Color...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="push_import">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Imports a tsv of experimental spectra to overlay on the plot.</string>
        </property>
        <property name="text">
         <string>&amp;Import...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="cb_import">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show imported data</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="push_colorBackground">
        <property name="text">
         <string>Set Color...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Foreground:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="push_colorForeground">
        <property name="text">
         <string>Set Color...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TODO</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="cb_calculate">
        <property name="text">
         <string>Show calculated spectra</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="push_font">
        <property name="text">
         <string>Change Font...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="9" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="10">
    <widget class="Avogadro::PlotWidget" name="plot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>800</width>
       <height>500</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Controls:
Double left click: Restore default axis limits
Right click + drag: Move plot
Middle click + drag: Zoom to region
Scroll wheel: Zoom to cursor</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QDoubleSpinBox" name="spin_FWHM">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>&amp;Gaussian Width:</string>
     </property>
     <property name="buddy">
      <cstring>spin_FWHM</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QComboBox" name="combo_yaxis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Transmittance (%)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Absorbance (%)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Y Axis Units:</string>
     </property>
     <property name="buddy">
      <cstring>combo_yaxis</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="spin_scale">
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>1.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sc&amp;ale:</string>
     </property>
     <property name="buddy">
      <cstring>spin_scale</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Infrared</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::PlotWidget</class>
   <extends>QFrame</extends>
   <header location="global">avogadro/plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>push_customize</tabstop>
  <tabstop>push_save</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>push_colorBackground</tabstop>
  <tabstop>push_colorForeground</tabstop>
  <tabstop>push_colorCalculated</tabstop>
  <tabstop>cb_labelPeaks</tabstop>
  <tabstop>cb_calculate</tabstop>
  <tabstop>push_colorImported</tabstop>
  <tabstop>push_import</tabstop>
  <tabstop>cb_import</tabstop>
  <tabstop>push_font</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SpectraDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
